Mathlib/CategoryTheory/Bicategory/Adjunction/Mate.lean

```diff
@@ -409,6 +439,79 @@ theorem conjugateEquiv_adjunction_id_symm {l r : c ⟶ c} (adj : l ⊣ r) (α :
   dsimp [Adjunction.id]
   bicategory
 
+end
+
+@[simp]
+lemma mateEquiv_leftUnitor_hom_rightUnitor_inv
+    {a b : B} {l : a ⟶ b} {r : b ⟶ a} (adj : l ⊣ r) :
+    mateEquiv adj adj ((λ_ _).hom ≫ (ρ_ _).inv) = (ρ_ _).hom ≫ (λ_ _).inv := by
+  simp only [← cancel_mono (λ_ r).hom, ← cancel_epi (ρ_ r).inv,
+    Category.assoc, Iso.inv_hom_id_assoc, Iso.inv_hom_id,
+    ← conjugateEquiv_id adj, conjugateEquiv_apply, Category.id_comp]
```

Comment:
```suggestion
  simp [← cancel_mono (λ_ r).hom, ← cancel_epi (ρ_ r).inv,
    ← conjugateEquiv_id adj, conjugateEquiv_apply]
```

[Apparently, we shouldn’t squeeze terminal simps](https://leanprover.zulipchat.com/#narrow/channel/270676-lean4/topic/Tool.20to.20convert.20.60simp.60.20to.20.60simp.20only.60/near/522999303)